.circleci/config.yml

```diff
@@ -97,6 +97,28 @@ jobs:
           paths:
             - ~/spree/vendor/bundle
 
+  brakeman:
+    <<: *defaults_3_0
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - spree-brakeman-{{ .Branch }}
+            - spree-brakeman
+      - run:
+          name: Ensure Brakeman Installed
+          command: |
+            gem install brakeman -v 4.3.1 && bundle install
```

Comment:
@aleksandarpetrushev please add this gem to `common_spree_dependencies`

Reply:
@damianlegawiec it is added, but when I try to execute with bundle, I get an error (https://app.circleci.com/pipelines/github/spree/spree/5491/workflows/3f2a5355-920e-4742-800a-97cffe8a22a6/jobs/48149). Same happens locally when I execute from repository root (with bundler). But when I just run brakeman --options (no bundle exec) it runs. Not sure how to go about this

Reply:
and same error without bundler on CircleCI

Reply:
@damianlegawiec it is definitely an incompatibility with brakeman versions and Rails/Ruby or other gems versions. It works with brakeman latest version.